distribution/docker/src/docker/Dockerfile

```diff
@@ -39,8 +39,8 @@ FROM centos:7
 
 ENV ELASTIC_CONTAINER true
 
-RUN for iter in {1..10}; do yum update -y && \
-    yum install -y nc && \
+RUN for iter in {1..10}; do yum update  --setopt=tsflags=nodocs -y && \
+    yum install -y  --setopt=tsflags=nodocs nc && \
```

Comment:
I am wondering how come this is needed as the base `centos:7` image contains this flag in `/etc/yum.conf`:

```
$ docker run --rm -ti centos:7 /bin/bash
[root@7969d2e1a52b /]# grep nodocs /etc/yum.conf
tsflags=nodocs
```